core/src/main/kotlin/ch/ergon/dope/resolvable/expression/unaliased/type/relational/ComparisonFunctionExpression.kt

```diff
@@ -0,0 +1,36 @@
+package ch.ergon.dope.resolvable.expression.unaliased.type.relational
+
+import ch.ergon.dope.resolvable.expression.UnaliasedExpression
+import ch.ergon.dope.resolvable.expression.unaliased.type.FunctionExpression
+import ch.ergon.dope.validtype.ComparableType
+
+sealed class ComparisonFunctionExpression<T : ComparableType>(
```

Comment:
Does this class provide any additional information? I would remove it and let the two classes below inherit directly from _FunctionExpression_

Reply:
I refactored the inheritances, now they inherit directly from it. Thanks!